.github/workflows/cd-syft-dev.yml

```diff
@@ -0,0 +1,167 @@
+name: CD - Syft - Deploy `dev` to K8s
+
+on:
+  # TODO: Re-enable once merged & tested
+  # schedule:
+  #   - cron: "0 */3 * * *"
+
+  workflow_dispatch:
+    inputs:
+      none:
+        description: "Run Tests Manually"
+        required: false
+
+jobs:
+  deploy-syft-dev:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Checkout Openmined/infrastructure
+        uses: actions/checkout@v4
+        with:
+          repository: OpenMined/infrastructure
```

Comment:
NIT: We might need to add a PAT token for infra repo, kindly check out stephen's PR: https://github.com/OpenMined/PySyft/pull/8065/files

Reply:
Not really needed as PATs are specific to a user's account. We should have a universal automation account (automation@openmined.org) that has enough permissions to perform these actions without the need for PATs.

Reply:
Resolved with @alfred-openmined-bot